executorlib/cache/queue_spawner.py

```
@@ -53,6 +54,7 @@ def execute_with_pysqa(
             "command": " ".join(command),
             "dependency_list": [str(qid) for qid in task_dependent_lst],
             "working_directory": os.path.abspath(cwd),
+            "submission_script_path": tempfile.NamedTemporaryFile().name,
```

Ruff

Use a context manager for opening files

(SIM115)